dataset.py

```diff
@@ -1,7 +1,20 @@
 import numpy as np
 from torch.utils.data.dataset import Dataset
 
+class to_spike_train(object):
+    def __init__(self, T):
```

Comment:
I agree with the idea, but so far I am not 100% sure how to implement that, for my purposes, I was fine with just passing in the T as the number of time steps I want to have. This class was also more an example of a possible transform, doesn't really have to included in the repo imo.

Reply:
I think that if it is included, it should be at the end of the file, so that the dataset is seen first by a reader, and then the optional transform. Maybe with an added comment, that this is an optional transform for the dataset to generate spiketrains or something like that

Reply:
Just pushed a commit where it's at the end. 👍🏻